tensorboard/components/tf_dashboard_common/tf-filterable-checkbox-list.html

```diff
@@ -61,12 +61,13 @@
       <template is="dom-repeat" items="[[_itemsMatchingRegex]]">
         <div class="item">
           <paper-checkbox
+            checked$="[[_isChecked(item, selectionState.*)]]"
             class="checkbox"
             name="[[item]]"
-            checked$="[[_isChecked(item, selectionState.*)]]"
-            on-change="_checkboxChange"
+            on-tap="_checkboxTapped"
+            title="ALT+Click to only select this item."
           >
-            <span>[[item.title]]</span>
+            <span title$="[[item.title]]">[[item.title]]</span>
```

Comment:
Does it add much to have the title in a tooltip as well as visibly shown?  I wonder if it would make more sense to set the title to the "ALT+Click" instruction to increase the ease of noticing it?